components/validator/config.go

```diff
@@ -144,11 +144,11 @@ func NewCLIConfig(ctx *cli.Context) CLIConfig {
 
 		// Optional Flags
 		AllowNonFinalized:            ctx.GlobalBool(flags.AllowNonFinalizedFlag.Name),
-		OutputSubmitterDisabled:      ctx.GlobalBool(flags.OutputSubmitterDisabledFlag.Name),
+		OutputSubmitterEnabled:       ctx.GlobalBool(flags.OutputSubmitterEnabledFlag.Name),
```

Comment:
Please move these required flags above `// Optional Flags`, and move `ProverGrpc` here.